doc/userguide/src/SupportingTools/Libdoc.rst

```diff
@@ -41,15 +41,22 @@ Synopsis
 Options
 ~~~~~~~
 
-  -f, --format <html|xml|xml:html>
-                           Specifies whether to generate HTML or XML output.
-                           `xml:html` format means generating an XML output
-                           where keyword documentation is converted to HTML
-                           regardless of the original documentation format. The
-                           default output format is got from the output file
-                           extension so that :file:`*.html` -> `html`,
-                           :file:`*.xml` -> `xml` and :file:`*.libspec`->
-                           `xml:html`.
+  -f, --format <html|xml> Specifies whether to generate an HTML or XML
+                          output file. The default output `format` and the 
+                          default output `specdocformat` is got from the
+                          output file extension.
+                          :file:`*.html`    -> `-f HTML -s HTML`
+                          :file:`*.xml`     -> `-f XML  -s RAW`
+                          :file:`*.libspec` -> `-f XML  -s HTML`
+   -s --specdocformat <raw|html>
+                          Specifies for XML outputs whether the keyword
+                          documentation is converted to HTML regardless
+                          of the original documentation format or kept
+                          raw as in the library or source spec file.
+                          :file:`*.html` files always get HTML documentation
+                          format. Default is based on the format.
+                          `--format HTML` -> `HTML`
+                          `--format XML`  -> `RAW`
```

Comment:
This doc is pretty confusing when it starts with saying this is related to XML outputs but then has an example related to HTML outputs at the end. It also doesn't explain that the default depends on the spec file extension. Finally, talking about spec files, not XML files, would also make this forwards compatible with JSON specs.

Perhaps the doc could be something like this:

```
Specifies the format of the documentation in generated spec file. Options:

- `raw`: Use the original documentation format. Default with `*.xml` spec extension.
- `html`: Convert documentation to HTML. Default with `*.libspec` spec extension.
```